### PR TITLE
enable Slovenian language

### PR DESCRIPTION
**Pull request by tmatek**
Should have run it locally 🙈

### CODE REVIEW

**Comment by OmGodse** [comment]
Can you tell why have you changed some of the translations?

**Comment by tmatek** [comment]
> Can you tell why have you changed some of the translations?

Sure! I ran the app in emulator and clicked through some of the menus in toolbar. I saw that a few translations could be improved.

File: app/src/main/res/values-sl/strings.xml [hunk 1]
```diff
@@ -68,6 +69,7 @@
     <string name="saved_to_notally">Shranjeno v Notally</string>
     <string name="empty_note">Prazen zapisek</string>
     <string name="empty_list">Prazen seznam</string>
+    <string name="select_labels">Izberi oznake</string>
```
Comment (tmatek) [inline app/src/main/res/values-sl/strings.xml:72]: This was missing (was in English).

File: app/src/main/res/values-sl/strings.xml [hunk 2]
```diff
@@ -7,16 +8,16 @@
     <string name="settings">Nastavitve</string>
 
     <!-- Menu Options -->
-    <string name="share">Pošlji</string>
```
Comment (tmatek) [inline app/src/main/res/values-sl/strings.xml:10]: This was translated as "Send", where it now reflects "Share".

File: app/src/main/res/values-sl/strings.xml [hunk 3]
```diff
@@ -7,16 +8,16 @@
...
+    <string name="share">Deli</string>
     <string name="filter">Filter</string>
     <string name="search">Iskanje</string>
     <string name="delete">Izbriši</string>
     <string name="restore">Obnovi</string>
-    <string name="archive">Arhiviranje</string>
```
Comment (tmatek) [inline app/src/main/res/values-sl/strings.xml:15]: This was in infinitive form.

**Pull request merged**